<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWidget</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>848</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="headerLayout">
     <item>
      <widget class="QToolButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset><normaloff>:filter-close</normaloff>:filter-close</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="chartTypeLabel">
       <property name="text">
        <string>Chart type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="chartType">
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="chartSubType">
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="firstAxisLabel">
       <property name="text">
        <string>First axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="firstAxis">
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="firstAxisBin">
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="firstAxisOperation">
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="secondAxisLabel">
       <property name="text">
        <string>Second axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="secondAxis">
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="secondAxisBin">
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="secondAxisOperation">
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="plot">
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="StatsView" name="stats">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatsView</class>
   <extends>QQuickWidget</extends>
   <header>stats/statsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../subsurface.qrc"/>
 </resources>
 <connections/>
</ui>
